.github/workflows/publish-new-release.yml

```diff
@@ -79,3 +79,35 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
```

Comment:
One concern to have this part of publish release - We are currently taking 2+hrs time to do an actual deployment from the time of release publish in github. 
If we can release within 15mins post release PR getting merged, this should be fine. cc: @ItsSudip

Reply:
In slack, we have option of schedule message but couldn't find in the plugin

Reply:
That would be redundant I believe notifying twice for the same release is incorrect.

Reply:
https://github.com/rudderlabs/rudder-devops/pull/5463

Reply:
Can we try out something like [this](https://app.slack.com/block-kit-builder/THZM86VSS#%7B%22blocks%22:%5B%7B%22type%22:%22header%22,%22text%22:%7B%22type%22:%22plain_text%22,%22text%22:%22:newspaper:%20%20Paper%20Company%20Newsletter%20%20:newspaper:%22%7D%7D,%7B%22type%22:%22context%22,%22elements%22:%5B%7B%22text%22:%22*November%2012,%202019*%20%20%7C%20%20Sales%20Team%20Announcements%22,%22type%22:%22mrkdwn%22%7D%5D%7D,%7B%22type%22:%22divider%22%7D,%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22%20:loud_sound:%20*IN%20CASE%20YOU%20MISSED%20IT*%20:loud_sound:%22%7D%7D,%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22Replay%20our%20screening%20of%20*Threat%20Level%20Midnight*%20and%20pick%20up%20a%20copy%20of%20the%20DVD%20to%20give%20to%20your%20customers%20at%20the%20front%20desk.%22%7D,%22accessory%22:%7B%22type%22:%22button%22,%22text%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Watch%20Now%22,%22emoji%22:true%7D%7D%7D,%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22The%20*2019%20Dundies*%20happened.%20%5CnAwards%20were%20given,%20heroes%20were%20recognized.%20%5CnCheck%20out%20*#dundies-2019*%20to%20see%20who%20won%20awards.%22%7D%7D,%7B%22type%22:%22divider%22%7D,%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22:calendar:%20%7C%20%20%20*UPCOMING%20EVENTS*%20%20%7C%20:calendar:%20%22%7D%7D,%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22%6011/20-11/22%60%20*Beet%20the%20Competition*%20_%20annual%20retreat%20at%20Schrute%20Farms_%22%7D,%22accessory%22:%7B%22type%22:%22button%22,%22text%22:%7B%22type%22:%22plain_text%22,%22text%22:%22RSVP%22,%22emoji%22:true%7D%7D%7D,%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22%6012/01%60%20*Toby's%20Going%20Away%20Party*%20at%20_Benihana_%22%7D,%22accessory%22:%7B%22type%22:%22button%22,%22text%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Learn%20More%22,%22emoji%22:true%7D%7D%7D,%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22%6011/13%60%20:pretzel:%20*Pretzel%20Day*%20:pretzel:%20at%20_Scranton%20Office_%22%7D,%22accessory%22:%7B%22type%22:%22button%22,%22text%22:%7B%22type%22:%22plain_text%22,%22text%22:%22RSVP%22,%22emoji%22:true%7D%7D%7D,%7B%22type%22:%22divider%22%7D,%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22:calendar:%20%7C%20%20%20*PAST%20EVENTS*%20%20%7C%20:calendar:%20%22%7D%7D,%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22%6010/21%60%20*Conference%20Room%20Meeting*%22%7D,%22accessory%22:%7B%22type%22:%22button%22,%22text%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Watch%20Recording%22,%22emoji%22:true%7D%7D%7D,%7B%22type%22:%22divider%22%7D,%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22*FOR%20YOUR%20INFORMATION*%22%7D%7D,%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22:printer:%20*Sabre%20Printers*%20are%20no%20longer%20catching%20on%20fire!%20The%20newest%20version%20of%20our%20printers%20are%20safe%20to%20use.%20Make%20sure%20to%20tell%20your%20customers%20today.%22,%22verbatim%22:false%7D%7D,%7B%22type%22:%22divider%22%7D,%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22Please%20join%20me%20in%20welcoming%20our%203%20*new%20hires*%20to%20the%20Paper%20Company%20family!%20%5Cn%5Cn%20*Robert%20California*,%20CEO%20%5Cn%5Cn%20*Ryan%20Howard*,%20Temp%20%5Cn%5Cn%20*Erin%20Hannon*,%20Receptionist%20%22%7D%7D,%7B%22type%22:%22divider%22%7D,%7B%22type%22:%22context%22,%22elements%22:%5B%7B%22type%22:%22mrkdwn%22,%22text%22:%22:pushpin:%20Do%20you%20have%20something%20to%20include%20in%20the%20newsletter?%20Here's%20*how%20to%20submit%20content*.%22%7D%5D%7D%5D%7D) for adding the features from the changelog?

Reply:
I couldn't find a way to get the changelog to add to the message.